beacon-chain/state/bellatrix.go

```diff
@@ -0,0 +1,6 @@
+package state
+
+// BeaconStateBellatrix has read and write access to beacon state methods.
+type BeaconStateBellatrix interface {
```

Comment:
No sure about introducing this type. It's just that returning `BeaconStateAltair` from `v3.InitializeFromProto` and `v3.InitializeFromProtoUnsafe` looks odd.

Reply:
Don't think this is true, we have a new exec payload method in bellatrix. cc @terencechain

Reply:
It's here: https://github.com/prysmaticlabs/prysm/blob/4906a0e6de91a4a3f768ff1d02bdb8d77235d810/beacon-chain/state/phase0.go#L55

Reply:
I'm sure there is a reason why it's in phase 0 code. Moving this method to Bellatrix causes issues in some interface usages. I am sure Terence has more answers.